modules.d/90multipath/multipathd-configure.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Device-Mapper Multipath Default Configuration
+Before=iscsi.service iscsid.service lvm2-activation-early.service
+Wants=systemd-udev-trigger.service systemd-udev-settle.service local-fs-pre.target
+After=systemd-udev-trigger.service systemd-udev-settle.service
```

Comment:
Are the deps on systemd-udev-*.service really needed? AFAICS `mpathconf --enable` doesn't access any HW; it just writes a config. file...